estate/models/estate.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models, fields
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate property model"
+
+    name = fields.Char('Property name', required=True, default="Unknown")
+    description = fields.Text('Property description')
+    postcode = fields.Char()
+    expected_price = fields.Float('Property expected price', required=True)
+    date_availability = fields.Date(default=fields.Date.today() + relativedelta(months=3), copy=False)
```

Comment:
```suggestion
    date_availability = fields.Date(default=fields.Date.add(fields.Date.today(), months=3), copy=False)
```
Your code is correct, but note that we have some Odoo built-in for that.
It mostly simplifies the imports 😉